Unable to resolve sugar-high module after installing with pnpm

I recently switched my project from npm to pnpm. After the switch, I (re)installed sugar-high using the `pnpm add sugar-high `command. The library was successfully added to the node_modules folder. However, when I tried to import and use the highlight function from sugar-high, the module could not be resolved.

As a temporary workaround, I've removed syntax highlighting from my project. However, having syntax highlighting working would be ideal.

Steps to Reproduce:

1. Switch the project from npm to pnpm (the node_module folder first get deleted then `pnpm i`
2. Install sugar-high using pnpm add sugar-high.
3. Try to import the highlight function from sugar-high.

Expected Behavior:

The sugar-high module should be resolved and the highlight function should be imported successfully.

Actual Behavior:

The sugar-high module could not be resolved, resulting in a build failure.

Error Message:
Type error: Cannot find module 'sugar-high' or its corresponding type declarations.

 ```
 3 | import { MDXRemote } from 'next-mdx-remote/rsc';
  4 | import { TweetComponent } from './tweet';
> 5 | import { highlight } from "sugar-high";
    |                           ^
  6 | import React from 'react';
  7 |
  8 | function Table({ data }) {
 ELIFECYCLE  Command failed with exit code 1.
```

Environment:

- Package Manager: pnpm
- Node.js version: v21.2.0
- Operating System: MacOS

Additional Context:

Is there a specific way to resolve this issue with pnpm? Any help would be greatly appreciated.

Thank you!

--- a/package.json
+++ b/package.json
@@ -2,6 +2,7 @@
   "name": "sugar-high",
   "version": "0.5.5",
   "type": "module",
+  "types": "./index.d.mts",
   "exports": {
     "types": "./index.d.mts",
     "default": "./lib/index.mjs"

